internal/rbd/rbd_util.go

```diff
@@ -264,7 +264,12 @@ func HexStringToInteger(hexString string) (uint, error) {
 
 // isKrbdFeatureSupported checks if a given Image Feature is supported by krbd
 // driver or not.
-func isKrbdFeatureSupported(ctx context.Context, imageFeatures string) bool {
+func isKrbdFeatureSupported(ctx context.Context, imageFeatures string) (bool, error) {
+	// return false when /sys/bus/rbd/supported_features is absent
+	// ex: kernel < v4.11.0
+	if krbdFeatures == 0 {
```

Comment:
if krbdFreatues is 0 just return false? as we don't need to support `tryOtherMounters` if we are not able to fetch the rbd features from the supported_features file.

Reply:
@pkalever looks like this is not addressed. can you please check it again?

Reply:
@Madhu-1 
Sorry! After all, I still strongly feel `isKrbdFeatureSupported()` should return an error when the file is not found instead of faking true/false. Can you just think over it once again, please?